starter-code/server.js

```diff
@@ -45,19 +61,28 @@ app.post('/articles', function(request, response) {
 
   function queryTwo() {
     client.query(
-      ``, // TODO: Write a SQL query to retrieve the author_id from the authors table for the new article
-      [], // TODO: Add the author name as data for the SQL query
+      `SELECT author_id
+      FROM authors
+      WHERE author = $1 ;`, // TODO: Write a SQL query to retrieve the author_id from the authors table for the new article
+      [request.body.author], // TODO: Add the author name as data for the SQL query
       function(err, result) {
         if (err) console.error(err)
+        console.log(result);
         queryThree(result.rows[0].author_id) // This is our third query, to be executed when the second is complete. We are also passing the author_id into our third query
       }
     )
   }
 
   function queryThree(author_id) {
     client.query(
-      ``, // TODO: Write a SQL query to insert the new article using the author_id from our previous query
-      [], // TODO: Add the data from our new article, including the author_id, as data for the SQL query.
+      `INSERT INTO articles (author_id, title, category, "publishedOn", body)
+      VALUES (${author_id}, $1, $2, $3, $4);`, // TODO: Write a SQL query to insert the new article using the author_id from our previous query
```

Comment:
Technically safe here, but do try to avoid any sort of string interpolation or string concatenation for SQL queries.